iptables/iptables.go

```diff
@@ -87,8 +87,14 @@ type IPTableEntry struct {
 	Params  string
 }
 
+type Client struct{}
+
+func NewClient() *Client {
+	return &Client{}
+}
```

Comment:
there are no fields (at all) being set in this struct, or other initialization magic - we don't need a constructor and can make the struct directly `... := &iptables.Client{}`